assets/sass/components/_programmes.scss

```diff
@@ -113,6 +113,237 @@
     }
 }
 
+/* =========================================================================
+   Process Map
+   ========================================================================= */
+
+.process-map {
```

Comment:
I'd probably put this in `_applications.scss` as it's more related to that, but pretty subjective so go with your gut.